### ISSUE TITLE
New email error

### ISSUE THREAD

**Issue by HeyAri** [HeyAri/Center-4-Certification#20]
Looking for certDate error listed on certificate.php page. 

Do we need to re-declare certDate or redo the date php in the emailAdmin File? 

Also it looks like I lost my cosmetic changes. Were they overwritten when I merged your last pull?

Notice: Undefined index: certDate in /home/heyari/public_html/center4certs/includes/emailAdmin.php on line 16

**Comment by HeyAri** [comment]
By the way, thanks again for all your help on this!

**Comment by shadowlion** [comment]
No problem - I don’t know which cosmetic changes were changed, but it might be helpful if you said something about them in the commits?

All the Best,

Justin Chiou
chiou.kai@gmail.com
630-204-8122
justinchiou.me

**Comment by HeyAri** [comment]
Sorry, Still learning the GitHub system, I only made a change to the css file that defines the certificate selected highlight border. I opened a pull request for it.


On Jun 19, 2017, at 10.41 A, Justin Chiou <notifications@github.com<mailto:notifications@github.com>> wrote:

No problem - I don’t know which cosmetic changes were changed, but it might be helpful if you said something about them in the commits?

All the Best,

Justin Chiou
chiou.kai@gmail.com<mailto:chiou.kai@gmail.com>
630-204-8122
justinchiou.me<http://justinchiou.me>
